atmosphere/cable.nml

```diff
@@ -55,7 +55,7 @@
   l_laiFeedbk   = .TRUE.  ! using prognostic LAI
   l_vcmaxFeedbk = .TRUE.  ! using prognostic Vcmax
 ! filenames for CASA-CNP input files
-!  casafile%cnpbiome='INPUT/CABLE-AUX-1.4/core/biogeochem/pftlookup_csiro_v16_17tiles.csv' ! biome specific BGC parameters
-  casafile%cnpbiome='INPUT/CABLE-AUX-1.4/core/biogeochem/pftlookup_csiro_v16_17tiles_wtlnds.csv'  ! biome specific BGC parameters
-  casafile%phen='INPUT/CABLE-AUX-1.4/core/biogeochem/modis_phenology_csiro.txt' ! phenology by latitude (modis derived)
+!  casafile%cnpbiome='INPUT/pftlookup_csiro_v16_17tiles.csv' ! biome specific BGC parameters
```

Comment:
```suggestion
```

This is a commented out line, so we should remove I think.

Reply:
Good point. Ive just removed it and done the same for the pre-industrial configuration